lib/datagrid/translation.rb

```diff
@@ -0,0 +1,25 @@
+require "action_view/helpers"
+
+module Datagrid
+  module Translation
+    include ActionView::Helpers::TranslationHelper
+
+    def translate_from_namespace(namespace, grid_class, key)
```

Comment:
I don't like this module idea. `Datagrid::Utils.translate_from_namespace` is better.